python/ray/tests/test_client_proxy.py

```diff
@@ -211,6 +211,31 @@ def raise_not_rewrite(input: JobConfig):
     server.stop(0)
 
 
+@pytest.mark.skipif(
+    sys.platform == "win32",
+    reason="PSUtil does not work the same on windows.")
+@pytest.mark.parametrize(
+    "call_ray_start", [
+        "ray start --head --ray-client-server-port 25031 "
+        "--port 0 --redis-password=password"
+    ],
+    indirect=True)
+def test_runtime_install_error_message(call_ray_start):
+    """
+    Check that an error while preparing the runtime environment for the client
+    server yields an actionable, clear error on the *client side*.
+    """
+    with pytest.raises(ConnectionAbortedError) as excinfo:
+        ray.client("localhost:25031").env({
+            "pip": ["anyscale-this-doesnt-exist"]
+        }).connect()
+    assert (
+        "No matching distribution found for anyscale-this-doesnt-exist" in str(
```

Comment:
```suggestion
            "pip": ["ray-this-doesnt-exist"]
        }).connect()
    assert (
        "No matching distribution found for ray-this-doesnt-exist" in str(
```

Maybe not `anyscale` here?  ;)